test/integration/shells_int_test.go

```diff
@@ -0,0 +1,99 @@
+package integration
+
+import (
+	"fmt"
+	"runtime"
+	"testing"
+
+	"github.com/ActiveState/cli/internal/testhelpers/e2e"
+	"github.com/ActiveState/cli/internal/testhelpers/tagsuite"
+	"github.com/stretchr/testify/suite"
+)
+
+type ShellsIntegrationTestSuite struct {
+	tagsuite.Suite
+}
+
+func (suite *ShellsIntegrationTestSuite) TestShells() {
+	suite.OnlyRunForTags(tagsuite.Critical, tagsuite.Shell)
+
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	var shells []e2e.Shell
+	switch runtime.GOOS {
+	case "linux":
+		shells = []e2e.Shell{e2e.Bash, e2e.Fish}
+	case "darwin":
+		shells = []e2e.Shell{e2e.Bash, e2e.Zsh, e2e.Tcsh}
+	case "windows":
+		shells = []e2e.Shell{e2e.Cmd}
+	}
+
+	// Checkout the first instance. It doesn't matter which shell is used.
+	cp := ts.SpawnWithOpts(e2e.WithArgs("checkout", "ActiveState-CLI/small-python"))
+	cp.Expect("Checked out project")
+	cp.ExpectExitCode(0)
+
+	for _, shell := range shells {
+		suite.T().Run(fmt.Sprintf("using_%s", shell), func(t *testing.T) {
+			// Run the checkout in a particular shell.
+			cp := ts.SpawnInShell(
+				shell,
+				e2e.WithArgs("checkout", "ActiveState-CLI/small-python", string(shell)),
+			)
+			cp.Expect("Checked out project")
+			if shell != e2e.Cmd {
+				cp.ExpectExitCode(0)
+			}
+
+			// There are 2 or more instances checked out, so we should get a prompt in whichever shell we
+			// use.
+			cp = ts.SpawnInShell(
+				shell,
+				e2e.WithArgs("shell", "small-python"),
+				e2e.AppendEnv("ACTIVESTATE_CLI_DISABLE_RUNTIME=false"),
+			)
+			cp.Expect("Multiple project paths")
+
+			// Just pick the first one and verify the selection prompt works.
+			cp.SendLine("\n")
```

Comment:
```suggestion
			cp.SendLine("")
```

This seems redundant.